### PR TITLE
dkms: add page

### PR DESCRIPTION
**Pull request by samuel-w**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/dkms.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+`dkms status`
+
+- Rebuild all modules for the currently running kernel:
+
+`dkms autoinstall`
+
+- Install version 1.2.1 of the acpi_call module for the currently running kernel:
+
+`dkms install -m acpi_call -v 1.2.1`
```
Comment (einverne) [inline pages/linux/dkms.md:15]:
```suggestion
`dkms install -m {{acpi_call}} -v {{1.2.1}}`
```

File: pages/linux/dkms.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Remove version 1.2.1 of the acpi_call module from all kernels:
+
+`dkms remove -m acpi_call -v 1.2.1 --all`
```
Comment (einverne) [inline pages/linux/dkms.md:19]:
```suggestion
`dkms remove -m {{acpi_call}} -v {{1.2.1}} --all`
```

**Review by einverne — CHANGES_REQUESTED** [review]
Thank you for the new page, I've left a comment for you to take a look.

File: pages/linux/dkms.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# dkms
+
+> A framework that allows for dynamic building of kernel modules.
```
Comment (navarroaxel) [inline pages/linux/dkms.md:3]:
```suggestion
> A framework that allows for dynamic building of kernel modules.
> More information: <https://github.com/dell/dkms>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by einverne — APPROVED** [review]

**Pull request merged**